pandas/core/frame.py

```diff
@@ -3485,8 +3499,30 @@ def unstack(self, level=-1):
         -------
         unstacked : DataFrame or Series
         """
-        from pandas.core.reshape import unstack
-        return unstack(self, level)
+        #from pandas.core.reshape import unstack
+        #return unstack(self, level)
+        result = self.T.stack(level, dropna=False, sequentially=sequentially).T
+
+        if isinstance(result, DataFrame):
```

Comment:
I don't think so. ``convert_objects()`` just works with object columns, but here I (may) need to convert float columns to int.